packages/app/src/specs/generators/ExpandableFileChooser.vue

```diff
@@ -54,9 +54,9 @@
       :search="filePathSearch"
     >
       <template #expanded-content="{file}">
-        <ReactComponentList
+        <slot
```

Comment:
I think we might lose some type safety here, but I'm not completely sure. This does make it more composable and generic, though.